app/user/rest.py

```diff
@@ -509,7 +509,7 @@ def set_permissions(user_id, service_id):
     # TODO fix security hole, how do we verify that the user
     # who is making this request has permission to make the request.
     service_user = dao_get_service_user(user_id, service_id)
-    user = service_user.user
+    user = get_user_by_id(user_id)
```

Comment:
Puzzling change here too, if we did remove the relationship to user from ServiceUser and then we make a separate db select to the database to get the user... why not dump dao_get_service_user? and use `dao_fetch_service_by_id`? 🤔